File: 백준/Silver/1065. 한수/한수.cc
#include <bits/stdc++.h>
using namespace std;

int a,rst;
string s;

int main() {
    ios::sync_with_stdio(false);
    cin.tie(NULL);

	cin >> a;
	int n = 1;
	while(n <= a){
		if(n<100) rst ++;
		else if(n>=100 && n<1000){
			s = to_string(n);
			if(s[1]-s[0]==s[2]-s[1]) rst++;
		}
		n++;
	}
	
	cout << rst;

    return 0;
}

File: 백준/Silver/1065. 한수/한수.py
a = input()

if int(a) < 100:
    rst = int(a)
    print(rst)

else :
    rst = 99
    for i in range(100,int(a)+1):
        n = str(i)
        if int(n[1])-int(n[0]) == int(n[2])-int(n[1]):
            rst += 1
    
    print(rst)